### DISCUSSION TITLE
Theme Change Not Working?

### DISCUSSION THREAD

**Discussion by nateProjects** [vkbo/novelWriter#1053] [Q&A] [answered]
Love novelWriter, but would also love to have dark mode working.

I can change the "Main GUI Theme" (and it keeps the setting).

But upon restart the writing window background it is still white.

Is my issue OS specific? Anything I can do to make it work?

Any help would be appreciated. Thanks.

I'm using Python 3.8.2 / MacOS 12.3.1 / Apple M1

_Note: I had to add `brew install pyqt@5` to get novel writer to work on my Mac._

**Comment by vkbo** [comment, marked as answer]
The GUI theme and the syntax highlighting theme are separate settings. The latter is in the "Highlighting" tab of Preferences. They are separate because there are a lot more syntax themes to choose from.

**Reply by nateProjects** [reply]
Ah. Doh. That fixed it. Appreciate it.

Not sure other people would easily pick up on that - then again it may just be me.

**Reply by vkbo** [reply]
You're right, people do not easily notice this. You're not the first to ask! The syntax theme setting was moved to its own tab as there ended up being a lot of settings for highlighting. I could add a label under the GUI theme setting saying this.

I've held off on redesigning this because I will move the whole theming feature into its own settings dialog as I want to let the user create custom themes themselves. See features #985 and #892.

**Comment by nateProjects** [comment]
Thanks for taking such an interest and being so responsive.

A label seems like a good solution until a theming tab is ready.

Cheers!
